src/controllers/aer_controller.hpp

```diff
@@ -1727,7 +1727,17 @@ Controller::simulation_methods(std::vector<Circuit> &circuits,
     bool superop_enabled = false;
     bool kraus_enabled = false;
     for (const auto& circ: circuits) {
-      auto method = automatic_simulation_method(circ, noise_model);
+      Controller::Method method;
+      try {
```

Comment:
Better to remove this try block and leave it as it was, and instead update the `automatic_simulation_method` at https://github.com/Qiskit/qiskit-aer/blob/872bdbaba177a4fb0eb2241b37dfcfac2233a79b/src/controllers/aer_controller.hpp#L1802 to return `return Method::statevector` at the end instead of throwing the the final exception which this added try block is always hiding.

If we get any other exceptions they should be legitimate errors which we want to throw normally.

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/1371/commits/42cbc7e72d302b3203c868d4459fa663c64161b5